.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,50 @@
+name: Lighthouse CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
I advice using tilde (`~`) sign to denote empty sections or marking it in other ways listed in [YAML specs](http://yaml.org/spec/1.2-old/spec.html#id2805071). Leaving label value empty may indicate that this section might be omitted by mistake.

Reply:
Done